server/src/main/java/io/unitycatalog/server/persist/MetastoreRepository.java

```diff
@@ -0,0 +1,19 @@
+package io.unitycatalog.server.persist;
+
+import java.util.UUID;
+
+/** The metastore is an object/resource to allow granting catalog-wide privileges. */
+public class MetastoreRepository {
+
+  private static final UUID METASTORE_ID = UUID.fromString("ca7a1095-537c-4f9c-a136-5ca1ab1ec0de");
```

Comment:
@vikrantpuppala FYI since you're working on metastore summary -- this needs to be updated.